agreements/financial.md

```diff
@@ -48,7 +50,7 @@ Money spent by the Foundation falls under the following categories.
 
 #### Fixed Costs
 
-The board is responsible for managing the fixed costs of the  from core funds, which are expected to be kept at ‘minimum viable’ to ensure smooth operations of core processes for the  network.
+The board is responsible for managing the fixed costs from core funds, which are expected to be kept at ‘minimum viable’ to ensure smooth operations of core processes for the network. The fixed costs are expected to cover catalysts' fees.
```

Comment:
@joshuavial rather than tack this onto fixed costs I'd suggest we break this out into something else like "Funding Financial Working Groups"

eg:

At its discretion the board can add/remove a Financially Enabled Working Group to its Fixed Costs table. When it does so The Board takes on the responsibility of holding the working group to account for funds spent.

Reply:
Agree with this, have amended.